server/src/main/java/org/apache/iotdb/db/mpp/plan/constant/StatementType.java

```diff
@@ -64,6 +64,7 @@ public enum StatementType {
   BATCH_INSERT_ROWS,
   BATCH_INSERT_ONE_DEVICE,
   MULTI_BATCH_INSERT,
+  LOAD_TSFILE,
```

Comment:
I think we should add this statement type to the INSERT_TIMESERIES privilege.

Reply:
After checking, I use MULTI_BATCH_INSERT statementType for LoadTsfileStatement.

Reply:
Yes, it would be better. BTW, should we delete 'LOAD_TSFILE' statementType here?

Reply:
Ok